src/content/developers/docs/scaling/zk-rollups/index.md

```diff
@@ -1,34 +1,206 @@
 ---
 title: Zero-Knowledge Rollups
-description: Introduction to zero-knowledge rollups
+description: An introduction to zero-knowledge rollups, a scaling solution used by the Ethereum community. 
 lang: en
 sidebar: true
----
+--- 
+
+Zero-knowledge rollups (ZK-rollups) are layer 2 [scaling solutions](/developers/docs/scaling/) that improve throughput on Ethereum Mainnet through off-chain execution. ZK-rollups process transactions independently, but post state data to Mainnet as `calldata`. They also produce cryptographic proofs of validity for transaction batches to guarantee the correctness of off-chain computation. 
 
 ## Prerequisites {#prerequisites}
 
 You should have a good understanding of all the foundational topics and a high-level understanding of [Ethereum scaling](/developers/docs/scaling/). Implementing scaling solutions such as rollups is an advanced topic as the technology is less battle-tested and continues to be researched and developed.
```

Comment:
I don't think this "prerequisites" statement is super helpful. It's not realkly necessary to understand "all foundational topics" to grasp the basics of ZK_rollups. Also, the second half of the sentence jars for me - basic topics can also be "less battle-tested" and under-researched. It is an advanced topic because it uses frontier technologies that are built on complicated cryptographic primitives. Better to keep it simple IMO.

```suggestion
You should have read and understood our page on [Ethereum scaling](/developers/docs/scaling/). 
```

Reply:
This is a good point. Although I should point out that other pages on scaling solutions (Plasma, Validium, Optimistic Rollups, etc.) use the same intro. I left it in since it was in the original page (wanted to focus on making what I felt were necessary changes). 

That said, I have PRs open for some of those pages on scaling. Do you want me to update the prerequisites section in those pages with your suggestion, too?